tests/components/airq/test_config_flow.py

```diff
@@ -48,7 +45,7 @@ async def test_form(hass: HomeAssistant) -> None:
 
     assert result2["type"] == FlowResultType.CREATE_ENTRY
     assert result2["title"] == TEST_DEVICE_INFO["name"]
-    assert result2["data"] == TEST_DATA_OUT
+    assert result2["data"] == TEST_USER_DATA
```

Comment:
Thanks for the suggestion. In this choice, I think I'd opt for a `SnapshotAssertion` to avoid replication dict content. (Although, I feel a bit reluctant to lose the explicit assertion of the few fields of the `FlowResult` which I expect to be changed by the tested functionality.)

Should it be a part of this PR?

Reply:
IMO Only the "data" key should be compared against the snapshot.
Other fields are OK as they are.
If can be done in another PR.